viddy/viddy.go

```diff
@@ -14,9 +14,11 @@ import (
 	"sync/atomic"
 	"time"
 
+	"github.com/adrg/xdg"
 	"github.com/gdamore/tcell/v2"
 	"github.com/moby/term"
 	"github.com/rivo/tview"
+	"github.com/spf13/viper"
```

Comment:
To reduce the dependency of the viddy lib, I think it will be great if we can remove dependency of the viper from lib.
What do you think?

Reply:
+1 on removing viper from the lib. assume it will stay in the TUI ... ?